ThunderCloud/Badge.swift

```diff
@@ -8,8 +8,66 @@
 
 import UIKit
 
+/// Keys for `Badge` properties from `Dictionary`
+/// TODO: Consider migration to `Codable`
+enum BadgeKey: String {
+    case completion
+    case how
+    case shareMessage
+    case title
+    case id
+    case icon
+    case campaign
+    case dateFrom
+    case dateUntil
+}
+
 /// `Badge` is a model representation of a storm badge object
-open class Badge: NSObject, StormObjectProtocol {
+open class Badge: NSObject, StormObjectProtocol
+{
```

Comment:
Just noticed this @BenCShutt! 😊 Can we keep these in-line with what we've currently got with the opening brace on the same line? 😀 Not sure if there's maybe a setting in Xcode for it?